### ISSUE TITLE
[Personal Cabinet] - provider block parent

### ISSUE THREAD

**Issue by litvinets** [ita-social-projects/OoS-Frontend#1193]
There is a new flag in application module:
![image](https://user-images.githubusercontent.com/27493959/169864041-7a81c6a5-6199-4fff-8bc4-318d48cf9804.png)
This flag can be changed by Api's
![image](https://user-images.githubusercontent.com/27493959/169864118-7859c207-b865-459e-9ab3-ad57d592d82b.png)
TODO:
1. Add new tab "Blocked" personal cabinet
2. Add status block to the cards and information about statuses (I think it is already done, you just need to uncomment it)
3. Add "three dots" to the application card (for PROVIDER view) where is an action to block/unblock user
4. Add Add checking if the card isBlock, if yes then displayed Block status.
5. Add pipe filterion for displaying all block cards in the tab
![image](https://user-images.githubusercontent.com/27493959/169873608-5ad2a715-ec5f-46d6-9ecb-e02d65c9bc2f.png)
6. display the reason for a block user with the same approach as for rejected status 
![image](https://user-images.githubusercontent.com/27493959/170025992-425102e8-728c-4296-afab-a7a56c464c2a.png)

https://github.com/ita-social-projects/OoS-Backend/issues/630
Additional info https://docs.google.com/spreadsheets/d/1o4rrCTvVfAEMTU2yKS0WJ0armgcVvBwNwfJwDW8lX30/edit#gid=0
